tests/e2e/tracer_shutdown/tracer_shutdown_test.py

```diff
@@ -0,0 +1,32 @@
+import os
+import signal
+import subprocess
+import time
+import unittest
+import uuid
+
+from autoblocks._impl.api.client import AutoblocksAPIClient
+
+client = AutoblocksAPIClient()
+
+
+class TestMainScript(unittest.TestCase):
+    def test_tracer_received_sigint_or_sigterm(self):
+        test_trace_id = uuid.uuid4()
+        # Start the main.py script as a subprocess
+        process = subprocess.Popen(["python", "tests/e2e/tracer_shutdown/main.py", str(test_trace_id)])
+        time.sleep(1)
```

Comment:
yeah since you're sleeping 1 here and only sleeping 1 in the evaluator it's likely the event was already sent before you even send the SIGTERM signal, which is why you need a longer sleep in your evaluator